docs/permission/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -10,31 +10,32 @@ For this tutorial, we'll use a basic permission check to authorize the `create`
 
 We'll start by creating a new permission, and then we'll use the permission api to call `authorize` with it during todo creation.
 
-## Setup
+## Creating a new permission
 
-Install the following module:
+Let's navigate to the file `plugins/todo-list-common/src/permissions.ts` and add our first permission:
 
-```
-$ yarn workspace @internal/plugin-todo-list-backend \
-  add @backstage/plugin-permission-common
+```diff
+  import { createPermission } from '@backstage/plugin-permission-common';
+
+- export const tempExamplePermission = createPermission({
+-   name: 'temp.example.noop',
+-   attributes: {},
++ export const todoListCreate = createPermission({
++   name: 'todo.list.create',
++   attributes: { action: 'create' },
+  });
 ```
 
-## Creating a new permission
+For this tutorial, we've automatically exported all permissions from this file (see `plugins/todo-list-common/src/index.ts`). For the actual plugins that you author, we recommend exporting all permissions from your plugin, so that Backstage integrators can import them when writing policies.
```

Comment:
```suggestion
For this tutorial, we've automatically exported all permissions from this file (see `plugins/todo-list-common/src/index.ts`). For the actual plugins that you author, you must all permissions from your plugin, so that Backstage integrators can import them when writing policies.
```